homeassistant/components/media_player/__init__.py

```diff
@@ -1164,7 +1164,7 @@ async def websocket_handle_thumbnail(hass, connection, msg):
         )
         return
 
-    await connection.send_big_result(
```

Comment:
We should drop this call as well as the above warning has been in place for a while